portsyncd/portsyncd.cpp

```diff
@@ -46,33 +46,33 @@ void handlePortConfigFromConfigDB(ProducerStateTable &p, DBConnector &cfgDb, boo
 
 int main(int argc, char **argv)
 {
-    Logger::linkToDbNative("portsyncd");
-    int opt;
-
-    while ((opt = getopt(argc, argv, "v:h")) != -1 )
+    try
     {
-        switch (opt)
+        Logger::linkToDbNative("portsyncd");
+        int opt;
+
+        while ((opt = getopt(argc, argv, "v:h")) != -1 )
         {
-        case 'h':
-            usage();
-            return 1;
-        default: /* '?' */
-            usage();
-            return EXIT_FAILURE;
+            switch (opt)
+            {
+            case 'h':
+                usage();
+                return 1;
+            default: /* '?' */
+                usage();
+                return EXIT_FAILURE;
+            }
         }
-    }
 
-    DBConnector cfgDb("CONFIG_DB", 0);
-    DBConnector appl_db("APPL_DB", 0);
-    DBConnector state_db("STATE_DB", 0);
-    ProducerStateTable p(&appl_db, APP_PORT_TABLE_NAME);
+        DBConnector cfgDb("CONFIG_DB", 0);
```

Comment:
Hi, these idents were done on purpose since the code was inserted under the "try" block, stated on line 49.

Reply:
Hi @prsunny , the PR merging is blocked due to coverage check failure (only 4 code lines are missing to reach the minimum threshold).
We've tried to expand existing unit tests or create new ones but found out that covering the missing lines via unit tests is very complex because it is called only from main() function.
Therefore, and because I didn't add any new functionality, I would ask for a waiver on the coverage check for this very unique case, Is it possible?

Reply:
> Hi @prsunny , the PR merging is blocked due to coverage check failure (only 4 code lines are missing to reach the minimum threshold). We've tried to expand existing unit tests or create new ones but found out that covering the missing lines via unit tests is very complex because it is called only from main() function. Therefore, and because I didn't add any new functionality, I would ask for a waiver on the coverage check for this very unique case, Is it possible?

@prsunny could you please refer to this question?